addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -1,4 +1,4 @@
-.o_hidden {
+:root .o_hidden {
```

Comment:
Note: if that kind of stuff is needed, it needs comments

Reply:
sure